websites/C/Canvas/metadata.json

```
@@ -14,7 +14,7 @@
 		"www.canvas.net"
 	],
 	"regExp": "((learn|www)[.]canvas[.]net|canvas[.]\\w+?([.]\\w+)+)|(\\w+?[.]instructure[.]com)",
-	"version": "1.0.1",
+	"version": "1.1.0",
 	"logo": "https://i.imgur.com/DRlJvzX.png",
```

Now that the presence is getting edited/updated, lets change the logo to 512x512
```suggestion
	"logo": "https://i.imgur.com/Bqsnmop.png",
```